### ISSUE TITLE
JSCP raises error

### ISSUE THREAD

**Issue by juli1** [kucherenko/jscpd#370]
**Describe the bug**

**To Reproduce**

Clone flask

```
git clone https://github.com/pallets/flask.git
```

Invoke JSCPD

```/usr/bin/jscpd -o output --min-lines 10 --reporters xml flask```

We have that

```
TypeError: Cannot read property 'rest' of undefined
    at Object.tokenize (/usr/lib/node_modules/jscpd/node_modules/reprism/lib/core.js:406:22)
    at tokenize (/usr/lib/node_modules/jscpd/node_modules/@jscpd/tokenizer/dist/tokenize.js:104:21)
    at Object.createTokenMapBasedOnCode (/usr/lib/node_modules/jscpd/node_modules/@jscpd/tokenizer/dist/tokenize.js:111:20)
    at Tokenizer.generateMaps (/usr/lib/node_modules/jscpd/node_modules/@jscpd/tokenizer/dist/index.js:12:27)
    at Detector.<anonymous> (/usr/lib/node_modules/jscpd/node_modules/@jscpd/core/dist/detector.js:33:46)
    at Generator.next (<anonymous>)
    at /usr/lib/node_modules/jscpd/node_modules/@jscpd/core/dist/detector.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/lib/node_modules/jscpd/node_modules/@jscpd/core/dist/detector.js:4:12)
    at Detector.detect (/usr/lib/node_modules/jscpd/node_modules/@jscpd/core/dist/detector.js:32:16)
```

**Expected behavior**

Expect JSCPD to finish

**Desktop (please complete the following information):**

 - NodeJS Version v10.21.0
 - jscpd version 3.3.0-rc.9

**Additional context**

When I invoke `jscpd` without any argument, I get the following error

```
jscpd         
TypeError: Cannot destructure property `path` of 'undefined' or 'null'.
    at detect (/usr/lib/node_modules/jscpd/node_modules/@jscpd/finder/dist/in-files-detector.js:41:39)
    at InFilesDetector.detect (/usr/lib/node_modules/jscpd/node_modules/@jscpd/finder/dist/in-files-detector.js:65:16)
    at Object.jscpd (/usr/lib/node_modules/jscpd/dist/index.js:41:25)
    at Object.<anonymous> (/usr/lib/node_modules/jscpd/bin/jscpd:15:12)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
```

**Comment by kucherenko** [comment]
thank you, investigating

**Comment by kucherenko** [comment]
fixed at jscpd@3.3.0-rc.10
